backend/README.md

```diff
@@ -1 +1,18 @@
-## Foodies Backend Project
\ No newline at end of file
+## Foodies Backend Project
+
+Requirements
+1. Postgres
+2. TypeORM Global Installation (`yarn add typeorm -g`)
+    - this will install the TypeORM CLI as well
+
+##### To import data from `init_data.sql`, run this command inside a command prompt. Make sure the schema is setup and it is currently empty
+
+> `psql -d foodies -f resources/init_data.sql --username={YOUR_USERNAME}`
```

Comment:
```suggestion
> `psql -d foodies -f src/resources/init_data.sql --username={YOUR_USERNAME}`
```

Just thought I'd add `src` assuming that people will be running this directly from within `backend/`

Reply:
Sounds good!